diff --git a/stream/dvb_tune.c b/stream/dvb_tune.c
index fa65156..53bacd0 100644
--- a/stream/dvb_tune.c
+++ b/stream/dvb_tune.c
@@ -18,7 +18,7 @@
 
    You should have received a copy of the GNU General Public License
    along with this program; if not, write to the Free Software
-   Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+   Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
    Or, point your browser to http://www.gnu.org/copyleft/gpl.html
 
 */

