diff -Naur mpv-0.14.0.orig/stream/dvb_tune.c mpv-0.14.0/stream/dvb_tune.c
--- mpv-0.14.0.orig/stream/dvb_tune.c 2015-12-11 21:20:29.000000000 +0300
+++ mpv-0.14.0/stream/dvb_tune.c      2015-12-12 22:49:55.059648377 +0300
@@ -18,7 +18,7 @@
 
    You should have received a copy of the GNU General Public License
    along with this program; if not, write to the Free Software
-   Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+   Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
    Or, point your browser to http://www.gnu.org/copyleft/gpl.html
 
 */

